numaprom/config/_config.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+from dataclasses import dataclass, field
+from omegaconf import MISSING
+
+from numalogic.config import NumalogicConf
+
+
+@dataclass
+class UnifiedConf:
+    unified_metric_name: str
+    unified_metrics: List[str]
+    unified_strategy: str = "max"
+    unified_weights: List[int] = field(default_factory=list)
+
+
+@dataclass
+class MetricConf:
+    metric: str = "default"
+    composite_keys: List[str] = field(default_factory=lambda: ["namespace", "name"])
+    static_threshold: int = 3
+    scrape_interval: int = 30
+    retrain_freq_hr: int = 8
+    resume_training: bool = False
+    numalogic_conf: NumalogicConf = MISSING
+
+
+@dataclass
+class NamespaceConf:
```

Comment:
since we could have more than one service/app in a namespace, should we include that also in the namespace field?

perhaps rename this as service conf and make namespace a mandatory field in that?

Reply:
Yes, I was debating on this. I thought of AppConf/ServiceConf instead of NamespaceConf, but currently metrics do not have app/service info so left it this way to do the testing.

But, yes service conf makes more sense than namespace level conf, will make the changes simultaneously in metrics too.

Reply:
I updated the class name to ServiceConf for now, will add the service or app variables once metrics have those details.